doc/source/rllib.rst

```diff
@@ -31,17 +36,23 @@ Then, you can try out training in the following equivalent ways:
 
 .. code-block:: bash
 
-  rllib train --run=PPO --env=CartPole-v0  # -v [-vv] for verbose,
-                                           # --config='{"framework": "tf2", "eager_tracing": true}' for eager,
-                                           # --torch to use PyTorch OR --config='{"framework": "torch"}'
+  rllib train --run=PPO --env=CartPole-v0 --stop='{"episode_reward_mean": 180.0}' --no-ray-ui
+      # -v [-vv] for verbose,
+      # --config='{"framework": "tf2", "eager_tracing": true}' for eager,
+      # --config='{"framework": "torch"}' to use PyTorch.
+
+Note that the `--no-ray-ui` tells the train script to run without the Ray dashboard.
+This option has been (soft) deprecated as of Ray 1.7 and you should now use the more
+explicit `--ray-ui` flag instead (in case you would like to run with the dashboard).
```

Comment:
hmm, why not just have the example show `--ray-ui`? Otherwise aren't you showing a deprecated flag?

Reply:
Showing `--no-ray-ui` would make the example work for any version of RLlib (also older ones).

Reply:
But it's a deprecated flag. If the user wants to use an older version of Rllib, shouldn't they read the correct version of the docs?